sparse/unit_test/Test_Sparse_IOUtils.hpp

```diff
@@ -0,0 +1,273 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include "KokkosKernels_IOUtils.hpp"
```

Comment:
```suggestion
#include "KokkosSparse_IOUtils.hpp"
```
I think the reason this didn't cause errors for you is it's included in a previous test header in Test_Sparse.hpp, but still good to make sure each test includes what it needs